src/solvers/dgsem_tree/indicators.jl

```diff
@@ -226,6 +226,52 @@ function Base.show(io::IO, ::MIME"text/plain", indicator::IndicatorMax)
   end
 end
 
+"""
+    IndicatorClamp(equations::AbstractEquations, basis; a=0.0, b=1.0, variable)
+    IndicatorClamp(semi::AbstractSemidiscretization; a=0.0, b=1.0, variable)
+
+A simple indicator returning 1.0 when the element average of `variable` lies within [a,b].
+Returns -1.0 otherwise.
+"""
+struct IndicatorClamp{RealT<:Real, Variable, Cache<:NamedTuple} <: AbstractIndicator
+  a::RealT
+  b::RealT
+  variable::Variable
+  cache::Cache
+end
+
+function IndicatorClamp(equations::AbstractEquations, basis; a = 0.0, b = 1.0, variable)
+  cache = create_cache(IndicatorClamp, equations, basis)
+  IndicatorClamp{typeof(a), typeof(variable), typeof(cache)}(a, b, variable, cache)
+end
+
+function IndicatorClamp(semi::AbstractSemidiscretization; a = 0.0, b = 1.0, variable)
+  cache = create_cache(IndicatorClamp, semi)
+  return IndicatorClamp{typeof(a), typeof(variable), typeof(cache)}(a, b, variable, cache)
+end
+
+function Base.show(io::IO, indicator::IndicatorClamp)
+  @nospecialize indicator # reduce precompilation time
+
+  print(io, "IndicatorClamp(")
+  print(io, "a=", indicator.a, "b=", indicator.b, "variable=", indicator.variable, ")")
```

Comment:
```suggestion
  print(io, "a=", indicator.a, ", b=", indicator.b, ", variable=", indicator.variable, ")")
```
To make the output easier to parse.

Reply:
Done.